pkg/monitor/cluster/cluster.go

```diff
@@ -73,6 +73,11 @@ func (mon *Monitor) Monitor(ctx context.Context) error {
 		return err
 	}
 
+	err = mon.emitNodesMetrics()
```

Comment:
I suspect this comment from @m1kola is also relevant here: https://github.com/Azure/ARO-RP/pull/334/files#r398434665

Reply:
@ehashman yes. I was wondering about the context cancellation too. I have checked some places where we use cli calls and none get ctx as a param. I'll check how to replace it